### PR TITLE
Feature/water model refactor

### PR DESCRIPTION
**Pull request by AHABID**

### CODE REVIEW

File: Pipe/schema.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
                     ],
                     "enum": [
                         "open",
```
Comment (bobeal) [inline Pipe/schema.json]: open should be OPENED ?

File: Pump/schema.json [hunk 2]
```diff
@@ -59,7 +59,8 @@
                     ],
                     "enum": [
                         "open",
-                        "close"
+                        "close",
```
Comment (bobeal) [inline Pump/schema.json]: OPEN, CLOSE instead ?

File: Valve/schema.json [hunk 3]
```diff
@@ -92,7 +92,8 @@
                     ],
                     "enum": [
                         "open",
-                        "close"
+                        "closed",
```
Comment (bobeal) [inline Valve/schema.json]: OPEN, CLOSED ?

File: Pipe/schema.json (hunk 1 above)
Comment (AHABID) [reply Pipe/schema.json]: just uppercase "OPEN"

File: Pump/schema.json (hunk 2 above)
Comment (AHABID) [reply Pump/schema.json]: OPEN, CLOSED and CV

File: Valve/schema.json (hunk 3 above)
Comment (AHABID) [reply Valve/schema.json]: OPEN, CLOSED and NONE

**Pull request merged**